glean-core/rlb/src/lib.rs

```diff
@@ -441,9 +443,60 @@ pub(crate) fn submit_ping_by_name_sync(ping: &str, reason: Option<&str>) {
     }
 }
 
+/// Indicate that an experiment is running.  Glean will then add an
+/// experiment annotation to the environment which is sent with pings. This
+/// infomration is not persisted between runs.
+///
+/// See `glean_core::Glean.set_experiment_active`
```

Comment:
@badboy , does this perform automatic linking in the future generated docs? Or should we copy/paste?

Reply:
```suggestion
/// See [`glean_core::Glean::set_experiment_active`].
```